frontend/src/services/eventHandlers/hubHandler.js

```diff
@@ -0,0 +1,12 @@
+import { GlobalStore } from "@/modules/common/store/globalstore";
+
+// Event handler for Hub API
+export default class HubHandler {
```

Comment:
what is hub handler?

Reply:
Its for handling websocket events from 'hub' api